.env

```diff
@@ -0,0 +1,3 @@
+localNodeAddress=http://localhost:18731/
+testNodeAddress=https://tezos-dev.cryptonomic-infra.tech/
```

Comment:
What is testNodeAddress?

Reply:
no need of env file anymore moved everything to config file just like cli